code/modules/client/preference/preferences.dm

```diff
@@ -289,6 +290,9 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 			if(species == "Grey")
 				dat += "<b>Voice:</b> <a href ='?_src_=prefs;preference=speciesprefs;task=input'>[speciesprefs ? "Wingdings" : "Normal"]</a><BR>"
 			dat += "<b>Secondary Language:</b> <a href='?_src_=prefs;preference=language;task=input'>[language]</a><br>"
+			if(species in list("Unathi", "Tajaran", "Plasmaman", "Kidan"))
```

Comment:
please don't add more of these awful "species in list" checks, they make everything harder to add new stuff to. Add a `var/datum/species/S = all_species[species]` to the top of here and check something like `if(istype(S) && S.autohiss_basic_map)`.